pkg/fixtures/workflows/github/kustomize/.github/workflows/azure-kubernetes-service-kustomize.yml

```diff
@@ -116,17 +119,17 @@ jobs:
           kustomizationPath: ${{ env.KUSTOMIZE_PATH }}
           kubectl-version: latest
         id: bake
-
       # Checks if the AKS cluster is private
       - name: Is private cluster
+        if: ${{ env.CLUSTER_RESOURCE_TYPE != 'Microsoft.ContainerService/fleets' }}
         id: isPrivate
         run: |
           result=$(az aks show --resource-group ${{ env.CLUSTER_RESOURCE_GROUP }} --name ${{ env.CLUSTER_NAME }} --query "apiServerAccessProfile.enablePrivateCluster")
           echo "PRIVATE_CLUSTER=$result" >> "$GITHUB_OUTPUT"
 
       # Deploys application based on manifest files from previous step
       - name: Deploy application
-        uses: Azure/k8s-deploy@v4
+        uses: Azure/k8s-deploy@v5.0.1
```

Comment:
you can use @v5 tag match new minor and patch versions instead of a specific patch version
`@v5` is equivalent to `@v5.*.*`, not `@v5.0.0` so it will use the patch version you want here